.github/workflows/ci-checks.yml

```diff
@@ -0,0 +1,109 @@
+name: CI Checks
+on:
+  push:
+    branches: [main]
+  pull_request:
+  workflow_dispatch:
+env:
+  IMAGE_NAME: "fruits-app"
+  BASE_IMAGE: "registry.access.redhat.com/ubi8/openjdk-11"
+  PROJECT_DIR: "fruits-app"
+  TARGET_DIR: target/
+  MVN_REPO_DIR: ~/.m2/repository
+
+jobs:
+  build-maven:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout Knative service deploy action github repository
+      - name: Checkout Knative service deploy action
+        uses: actions/checkout@v2
+        with:
+          path: "kn-service-deploy"
+
+      # Checkout example repository
+      - name: Checkout example code
+        id: git_checkout
+        uses: actions/checkout@v2
+        with:
+          repository: kameshsampath/kn-service-action-example
```

Comment:
we should move this into our org at some point. probably we'll have to consolidate our samples into one repo.

Reply:
Yeah, or else later I will change the example to point to some official one